distribution/lib/Standard/Base/0.0.0-dev/src/Runtime/Managed_Resource.enso

```diff
@@ -44,6 +44,17 @@ type Managed_Resource
        e.g. using `Managed_Resource.register cache cleanup_fn True` is useful
        for creating user managed caches.
 
+       ! Multiple `Managed_Resource`s for the same resource
+
+         It is considered an error to use the same resource in multiple
+         `Managed_Resource`s, since each `Managed_Resource` has its own,
+         possibly different, finalizer.
+
+         Note that for truly atomic values such as integer `2`, all instances
```

Comment:
Associating `Managed_Resource` with primitive type is error on its own:
- Panic if the resource isn't `TruffleObject`. 
- Rephrase this paragraph saying _integer_ cannot be managed resource

Reply:
>     * Panic if the resource isn't `TruffleObject`.

d85b896ee0 shows how such a check for `TruffleObject` might look like. It changes the behavior of tests using `2` as a resource. Feel free to remove d85b896ee0 commit, if you aren't OK with it.